src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -78,7 +56,7 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
 
     $ids = [];
 
-    if ($this->configuration['handler_settings']['field_mode'] == 'admin') {
+    if ($this->isGroupAdmin()) {
```

Comment:
Sure. In D7 we mock the instance to have this dummy value, so we could render the widget based on is it "my groups" or "other groups". Quick call on Monday?